canvas_cli/utils/validators/manifest_schema.py

```diff
@@ -104,11 +104,10 @@
                             "read": {"type": "array", "items": {"type": "string"}},
```

Comment:
do I understand correctly that we're leaving `data_access` here because plugins that already include it would break?

Reply:
Yup, that is my intention -- to let it peacefully exist if it's there, but not encourage its usage going forward